nrf_security/Kconfig

```diff
@@ -0,0 +1,468 @@
+menu "Nordic Security"
+
+config NORDIC_SECURITY_BACKEND
+	bool
+	prompt "Use Nordic provided security backend"
+	depends on SOC_FAMILY_NRF
+	select NRFXLIB_CRYPTO
+	help
+	  Help message
+
+if NORDIC_SECURITY_BACKEND
+
+comment "Backend Selection"
+
+config CC310_BACKEND
+	bool "Use cc310 backend"
+	default y
+	depends on SOC_NRF52840
+	select NRF_CRYPTO_MULTIPLE_CHOICE_GROUP_0 if VANILLA_BACKEND
+	help
+	  Enable cc310 backend
+
+config VANILLA_BACKEND
```

Comment:
"vanilla" is perhaps too idiomatic. How about either of these in place of vanilla/VANILLA?

 - mbedtls
 - software or sw
 - standard

Reply:
We have not found a better term to describe the fact that the mbedtls is in an unaltered, de-facto state. This is the reason for the somewhat idiomatic term. 

And we wanted to shy away from SW as part of the CC310 functionality is based on running some software on top if only parts of the algorithm is HW accelerated.

Reply:
Fair enough. Take care that it isn't interpreted as a proper noun "Vanilla".

Reply:
We will amend the name to include mbedtls for clarity